js/components/controlBar.js

```diff
@@ -502,14 +519,19 @@ var ControlBar = React.createClass({
     this.moreOptionsItems = collapsedMoreOptionsItems;
 
     finalControlBarItems = [];
-
+     
     for (var k = 0; k < collapsedControlBarItems.length; k++) {
       if (collapsedControlBarItems[k].name === "moreOptions" && (this.props.controller.state.isOoyalaAds || collapsedMoreOptionsItems.length === 0)) {
         continue;
       }
-
-      finalControlBarItems.push(controlItemTemplates[collapsedControlBarItems[k].name]);
-    }
+      var position = 'center';
+      if (k === collapsedControlBarItems.length -1) {
```

Comment:
It adds right alignment for the last item of the control bar, left alignment for the first item and center alignment for the rest of the items. Alignment is the position of the tooltip's pointer (small triangle under the box)